src/Build/Evaluation/Evaluator.cs

```diff
@@ -201,11 +196,29 @@ internal class Evaluator<P, I, M, D>
             int submissionId,
             EvaluationContext evaluationContext,
             bool profileEvaluation,
-            bool interactive)
+            bool interactive,
+            ILoggingService loggingService,
+            BuildEventContext buildEventContext)
```

Comment:
You have to update the callsites to pass in this information right?

Reply:
Ah, this is private, nevermind :)

---
In reply to: [296953080](https://github.com/microsoft/msbuild/pull/4461#discussion_r296953080) [](ancestors = 296953080)